AttributeError: module 're' has no attribute 'Match'

**Issue:**

Running `make` command in root directory with any arguments fails with a Python `AttributeError`.

**Environment:**

Bash REPL on [`replit.com`](https://replit.com/new/bash). Cloned `polygott` repo into base directory.

**Error:**
```py
Traceback (most recent call last):
  File "./extra/manifest_tool.py", line 465, in <module>
    _main()
  File "./extra/manifest_tool.py", line 459, in _main
    context(args)
  File "./extra/manifest_tool.py", line 333, in context
    def _sub_globs(match: re.Match) -> str:
AttributeError: module 're' has no attribute 'Match'
Makefile:30: recipe for target 'build/context.tar.bz2' failed
make: *** [build/context.tar.bz2] Error 1
```

--- a/extra/manifest_tool.py
+++ b/extra/manifest_tool.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/env python3
 """Tool to help splitting and splicing docker image layers.
 
 There are four subcommands:

